ArmRobot/Assets/Scripts/DomainRandomization.cs

```diff
@@ -17,8 +17,8 @@ public class DomainRandomization : MonoBehaviour
     public int nbMaxDistractorObjects = 8;
     public float rotationAngle = 10.0f;
 
-    public float yAltitudeTable = 0.813f;
-    public float minimumDistanceWithoutObjects = 0.05f;
+    public float yAltitudeTable = 8.13f;
+    public float minimumDistanceWithoutObjects = 0.5f;
```

Comment:
You could just scale the numbers at the time of data collection, you don't actually have to change the scene in order to scale the label data as we talked about.

Reply:
Yes that is true but even for the further experiment it will be easier to have everything at scale 1